helix-term/src/ui/prompt.rs

```diff
@@ -336,10 +335,7 @@ impl Prompt {
 
         let (range, item) = &self.completion[index];
 
-        // since we are using shellwords to parse arguments, make sure
-        // that whitespace in files is properly escaped.
-        let item = shellwords::escape(item);
-        self.line.replace_range(range.clone(), &item);
+        self.line.replace_range(range.clone(), item);
```

Comment:
Should we pass the entire item instead of just the reference?

Reply:
`item` is already a `&Cow<str>` here. The old code was taking a reference because `shellwords::escape` returns a `Cow<str>`

Reply:
Got it. LGTM